src/scripts/drag-text.js

```diff
@@ -117,6 +118,11 @@ H5P.DragText = (function ($, Question, ConfirmationDialog) {
     // Keeps track of if Question has been answered
     this.answered = false;
 
+    // If NumLines Parameter is properly set this will select a random set of paragraphs from the base textField parameter	  
+    if (!(this.params.behaviour.numLines <= 2 || this.params.behaviour.numLines >= this.params.textField.split('\n\n').length)) {
```

Comment:
1. Doesn't `\n\n` imply that you will have to have a blank line in between lines with text? It may not be obvious to an author that you mean a blank line in between to signal a border between paragraphs.
2. What if the author adds more blank lines?

Reply:
1. Yes, that was a consideration, originally I had it splitting at `/n` but in cases where there are multiple lines that are relevant together it is useful to divide `/n/n`. However, I do realize that could be confusing to users and probably needs refinement
2. Good point. I'll make a work around for that